python/foglamp/services/core/routes.py

```diff
@@ -91,6 +92,9 @@ def setup(app):
     app.router.add_route('POST', '/foglamp/service', service.add_service)
     app.router.add_route('GET', '/foglamp/service', service.get_health)
 
+    # South
+    app.router.add_route('GET', '/foglamp/south', south.get_south_services)
```

Comment:
I would suggest if this is getting services in the  south then it would be better named as /foglamp/service/south so that it is clear it is service related.

Reply:
Instead of creating new route, will be better if we could handle with an optional param `type` in foglamp.services.core.api.service.get_health.   i.e `curl -X GET http://localhost:8081/foglamp/service?type=Southbound | jq`

Reply:
Yes, sure. Please can you talk with Amarendra about the change he is proposing. I think that what he wants can also be obtained via this mechanism if we include the schedule in the health of a service.

Reply:
I kept this specifically for `/south` and `/north` needs in mind. 

Northbound tasks are not services, but that need an aggregated fetch for all north schedules, its enabled/disabled status, sent stats.

Reply:
True, but we are going to end up with a very strange API here because of the way we are mapping this articular client to the API. In order to move forward lets keep whatever works but as undocumented API until we decide what we really want.